Lib/test/test_turtle.py

```diff
@@ -526,6 +546,97 @@ def test_save(self) -> None:
             with open(file_path) as f:
                 assert f.read() == "postscript"
 
+    def test_no_animation_sets_tracer_0(self):
+        s = turtle.TurtleScreen(cv=unittest.mock.MagicMock())
+
+        with s.no_animation():
+            assert s.tracer() == 0
```

Comment:
Please use the `unittest` assert methods iso. the `assert` keyword. Ditto for the rest of the added tests.